grpc-server/src/main/java/com/netflix/conductor/grpc/server/GRPCServer.java

```diff
@@ -21,9 +18,9 @@
 
     private final Server server;
 
-    public GRPCServer(int port, BindableService... services) {
+    public GRPCServer(int port, List<BindableService> services) {
```

Comment:
**Question:** Why the change to a `List` from varargs?

Reply:
Only consumer of this constructor was `GRPCServerProvider`. As I needed to add some optional bindable services, the question was more related to choose one of these options:
- Updating GRPCServer to take a list of BindableServices(GRPCServer change)
- Converting to array the BindableService list (GRPCServerProvider change).
I chose to make the change on GRPCServer because:
- I thought that converting to array the BindableService list on GRPCServerProvider was unnecessary as there was only one use of the constructor.
- I thought we were not losing in readability that much